sdks/java/io/file-based-io-tests/pom.xml

```diff
@@ -294,6 +294,12 @@
                     <version>${apache.hadoop.version}</version>
                     <scope>runtime</scope>
                 </dependency>
+                <dependency>
+                    <groupId>javax.xml.bind</groupId>
+                    <artifactId>jaxb-api</artifactId>
```

Comment:
Please add this to root level dependencyManagement so that we use the same version of jaxb-api across components.

Reply:
Sorry, by root level I meant beam/pom.xml

Reply:
Added to dependencyManagement from java/io. Version removed from filebased pom.xml as because it's defined in dependencyManagement and this one will be used. To be consistent with scopes in dependencyManagement scope was also changed to test and it was sufficient to run tests.

Reply:
No problem, I will move it to beam/pom.xml, but I need to run local tests, before another pull.

Reply:
Thanks. Can you also define JAXB dependencies in https://github.com/apache/beam/blob/master/sdks/java/io/xml/pom.xml in root level and update that component to use the version defined in the root level. Also, can't we use 2.2.3 instead of 2.2.2 which seems to be pretty old ? I think we could run into issues if we have to use both 2.2.0 and 2.2.3 for different components.